rpc/wrtc_peer.go

```diff
@@ -207,6 +207,8 @@ func newPeerConnectionForServer(
 	if err != nil {
 		return pc, dataChannel, err
 	}
+	defer utils.UncheckedError(negotiationChannel.Close())
```

Comment:
[major] this should only close on peer/clientconn close. We use this negotiation channel actively throughout the connection

Reply:
oh whoops; I thought this channel wasn't needed beyond `newPeerConnectionForServer`. I'm not sure we should `Close` it at all then. the modified `initialDataChannelOnError` function already ignores the "User Initiated Abort: Close called" error, so the "premature" error is still gone even without this deferred close. I could write a new `negotiationChannelOnError` function that can close the negotiation channel on reception of "User Initiated Abort: Close called" but up to you—I'm a little out of my depth

Reply:
yeah that's fine to not close then